index.html

```diff
@@ -1,58 +1,32 @@
+
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Awesome Books</title>
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
-    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Awesome Book</title>
 </head>
-<body>
-    <section class="books-section">
-        <h1 class="heading">AWESOME BOOKS</h1>
-         <div class="books-li">
-            <!-- <div class="the-book">
-             <ul class="list-unstyled">
-                <li class="titles">Lorem ipsum</li>
-                <li class="book-author">Testeroo Testyy</li>
-            </ul>
-            <button class="btn-remove">Remove</button>
-            </div>
 
-            <hr>
-
-            <div class="the-book">
-                <ul class="list-unstyled">
-                    <li class="tittles">second book</li>
-                    <li class="book-author">Testeroo Testyy</li>
-                    
-                </ul>
-                <button class="btn-remove">Remove</button> -->
-
-               
-            </div>
-            <hr> 
-            </div>
-     
-    </section>
-    <form id="form">
-        <div>
-          <div>
-          <label for="title"></label>
-          <input type="name" required id="title" name="title" placeholder="title" >
-          </div>
-          <div>
-           <label for="Author"></label>
-           <input type="name" required id="author" name="author" placeholder="author" >
-          </div>
-        <div>
-          <button id="add-btn" type="submit">
-             Add
-          </button>
-          <span></span>
+<body>
+    <section class="main-container">
```

Comment:
Great job implementing the functionalities 👍, According to the wireframe, you are to add a navigation bar. Kindly add it to meet up with the requirement. 

| Your Design | Wireframe Design |
| :---:   | :---: | 
| <img width="958" alt="image" src="https://user-images.githubusercontent.com/73854999/206857399-7bfb41f7-6777-4478-ad23-d8310ae5cd6c.png">|<img width="298" alt="image" src="https://user-images.githubusercontent.com/73854999/206857377-8344d55a-c27a-4756-a263-aeca30f3c596.png">|

Kindly refer back to the wireframe and add the necessary pages.